app/src/main/java/org/mozilla/rocket/home/HomeFragment.kt

```diff
@@ -500,6 +501,14 @@ class HomeFragment : LocaleAwareFragment(), ScreenNavigator.HomeScreen {
         homeViewModel.showContentHubClickOnboarding.observe(viewLifecycleOwner, Observer { couponName ->
             showRequestClickContentHubOnboarding(couponName)
         })
+        homeViewModel.showKeyboard.observe(viewLifecycleOwner, Observer {
+            Looper.myQueue().addIdleHandler {
+                if (!isStateSaved) {
```

Comment:
why do we need to check the `isStateSaved`?

Reply:
In case user tap back leave the home fragment right after clicking new tab button in tab tray.